### PR TITLE
Add a configuration for csv export

### PR DESCRIPTION
**Pull request by AdJez**
Hi !
First, thanks a lot for this project it's nicely written and saves a lot of time!

I have just added a little provider for csv exportation. I thought it might be useful for others. Tell me what you think about it !

Adrien Jézégou

### CODE REVIEW

File: src/javascripts/ng-admin/Crud/button/CsvExportConfiguration.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+define(function () {
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/button/CsvExportConfiguration.js]: Why put that under the button dir?

Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/button/CsvExportConfiguration.js]: We're not using requirejs anymore, but webpack. Please leave the `define` aside and use `export` instead.

**Comment by fzaninotto** [comment]
Great idea! However, I think the implementation should be angular-independant (and therefore made inside `admin-config` instead of `ng-admin`). We're trying to abstract the implementation away from the configuration, which allows us to implement [react-admin](https://github.com/marmelab/react-admin) using the exact same configuration as ng-admin.

Would you like to try and make the changes in `admin-config` instead?

**Comment by AdJez** [comment]
Thanks for this quick response, i'll make this changes into admin-config instead of ng-admin and send you once done.

**Comment by jpetitcolas** [comment]
Any news on it? :)

**Comment by fzaninotto** [comment]
Superseded by #933

**Pull request closed without merging**